workshops/build-a-binderhub/before_workshop.md

```diff
@@ -1,2 +1,26 @@
-A place to store the email sent out to attendees before the workshop.
-See #222 for a draft of the content.
+Hi!
+
+We are looking forward to welcoming you to the 'Build a Binderhub' workshop in Sheffield next week! We have a couple of things for you to look at before the workshop and some more information about what is happening on the day:
+
+## Getting ready
+We have funds available for travel and childcare support. Please get in touch with theturingway@gmail.com to discuss how we can support you.
+
+Before you arrive please make sure you have the following:
+* A “Free Trial” subscription on Microsoft Azure or an existing Azure account with an active subscription you would like to use
```

Comment:
Can you add a link to where to get the free azure account? Cc @sgibson91

Reply:
Is this the right one @sgibson91 https://azure.microsoft.com/en-gb/free/ ?

Reply:
Yep, that's the one!

Reply:
Thanks!